templates/set-lcm-mode.ps1.erb

```diff
@@ -0,0 +1,51 @@
+Write-Host "Setting LCM"
+$LCMRefreshMode = '<%= @refresh_mode %>'
+$generationDate = (get-date -format "M/d/yyyy HH:mm:ss")
+$encoding       = [system.Text.Encoding]::UTF8
```

Comment:
Really? I have always followed shortened on the command line, fully qualified (full namespace, no aliases, etc...) in the script.